opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/endpoints/OpenSearchEndpoint.java

```diff
@@ -14,6 +14,32 @@
  **/
```

Comment:
So there should be no changes to the OpensearchEndpoint right? You could just checkout these changes so we aren't just reorganizing the imports.